packages/esm-patient-chart-app/src/visit/visit-prompt/end-visit-dialog.component.tsx

```diff
@@ -16,13 +16,29 @@ const EndVisitDialog: React.FC<EndVisitDialogProps> = ({ patientUuid, closeModal
   const { currentVisit, currentVisitIsRetrospective, mutate } = useVisit(patientUuid);
   const { queueEntry } = useVisitQueueEntry(patientUuid, currentVisit?.uuid);
 
+  const calculateStopDatetime = (): Date => {
+    const MinVisitdurMs = 1000;
```

Comment:
* Variables should be something like `minVisitDurationMs`
* We shouldn't be hard-coding this at all
* I don't think this helps solve the problem in the ticket which is starting a new visit after ending the current one. This makes an adjustment to the visit stop date based on the _current_ visit start date, but the visit start date we care about is the _new_ visit. The form itself should already enforce the rule that `stopTime >= startTime` for a visit.